sirepo/pkcli/admin.py

```diff
@@ -49,20 +49,22 @@ def _audit_user(uid, proprietary_sim_types):
                 )
 
     def _link_or_unlink_proprietary_files(sim_type, should_link):
-        b = sim_data.get_class(sim_type).FLASH_RPM_FILENAME
-        p = simulation_db.simulation_lib_dir(sim_type).join(b)
-        if not should_link:
-            pkio.unchecked_remove(p)
-            return
-        try:
-            s = proprietary_code_dir(sim_type).join(b)
-            assert s.check(file=True), f'{s} not found'
-            p.mksymlinkto(
-                s,
-                absolute=False,
-            )
-        except py.error.EEXIST:
-            pass
+        for f in proprietary_code_dir(sim_type).listdir(
```

Comment:
I think this is simpler: `pkio.sorted_glob(proprietary_code_dir(sim_type).join('*'))`  Maybe add `pkio.sorted_list_dir()`? I don't think you need to care about whether something is a file or a directory.

Reply:
I made the change to pkio.sorted_glob. I enforced that it had to be a file because of [this (comment)](https://github.com/radiasoft/sirepo/pull/2490#discussion_r423821484).

Reply:
https://github.com/radiasoft/sirepo/pull/2490/commits/cd07af6a29565ca8f4b5a3144ff997244442b7ea